index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Survey | Form</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+        <h1 id="title">Microverse Trials session</h1>
+        <p id="description">this session is done by Eric and Toka</p>
+      </header>
```

Comment:
Please you should use the proper way of semantic tags. the <header> should be above the <div> tag please you should change that.

Reply:
Good job 👍 in identifying the proper use of semantic tags.